app/elements/socobo-inventory/socobo-inventory-validate-behavior.html

```diff
@@ -37,14 +37,21 @@
       },
 
       /**
-       * Possible categories the item can be associated with
+       * Quick select options for the expiry date
        *
        * @private
        */
-      categories: {
+      dateOffsets: {
         type: Array,
         value: function() {
-          return ["Backen", "Gewürze", "Gemüse", "Obst"];
+          return [
+            this.Offset("none", 0, "day"),
```

Comment:
But that would make no sense. `None` should set the expiry date to *none*. That is, clear the input.